lib/jhipsteruml.js

```diff
@@ -36,12 +37,14 @@ if (options.displayVersion) {
   process.exit(0);
 }
 
-if (!fs.statSync('./.yo-rc.json').isFile()) {
-  console.info(
+fs.exists('./.yo-rc.json', function (exists) {
```

Comment:
Didn't get it working outside of a JHipster project like that.
Agree, try, catch like [L49l](https://github.com/bmoers/jhipster-uml/blob/89a5335c5cad873c7225c4bb980c5ffe099ad688/lib/jhipsteruml.js#L49l) would have been better.